atc/engine/step_factory.go

```diff
@@ -77,7 +80,7 @@ func (factory *coreStepFactory) GetStep(
 		containerMetadata,
 		factory.lockFactory,
 		factory.resourceCacheFactory,
-		factory.strategy,
```

Comment:
I believe this one is using the default strategy `volume-locality`

Reply:
Yes. But as I explained in the other comment, `volume-locality` doesn't work for `get`.